src/BinSkim.Rules/RuleIds.cs

```diff
@@ -71,5 +71,8 @@ internal static class RuleIds
         // MachO Checks
         public const string EnablePositionIndependentExecutableMachO = "BA5001";
         public const string DoNotAllowExecutableStack = "BA5002";
+
+        // Performance (not security) checks
+        public const string DisableIncrementalLinking = "BA6001";
```

Comment:
>DisableIncrementalLinking [](http://example.com/codeflow?start=28&length=25)

OK, here comes the quibbling with semantics. :) Incremental linking, in fact, needs to be explicitly opted in, and so 'disabling' it refers to the absence of a specific action. And so I'd reverse it, and frame the rule to advise against opting into this adverse condition. Also might want to qualify the name so that it's clear we're talking release builds.

Note this readable rule name serves as a nice abstract for the check, with useful detail, still pretty concise. That's the goal.

DoNotEnableIncrementalLinkingInReleaseBuilds

Reply:
That's more specific so it seems good.  Will change.